lib/ex_unit/lib/ex_unit.ex

```diff
@@ -211,6 +213,9 @@ defmodule ExUnit do
       on formatting and reporters (defaults to 20)
 
     * `:timeout` - set the timeout for the tests (default 60_000ms)
+
+    * `:plural_rules` - See `ExUnit.plural_rule/1` and `ExUnit.plural_rule/2`.
```

Comment:
Let's remove this from here since you are not supposed to set it directly anyway.